.travis.yml

```diff
@@ -13,6 +13,7 @@ before_install:
     - source activate test
     - travis_retry conda install --yes pytest
     - conda info -a
+    - pip install -r requirements-dev.txt
```

Comment:
@tomkralidis can you "fold" the `pytest` into `requirements-dev.txt`, remove the `pytest` line above and install `requirements-dev.txt` wth conda?